package.json

```diff
@@ -95,5 +96,8 @@
   "volta": {
     "node": "18.12.1",
     "yarn": "1.22.19"
+  },
+  "peerDependencies": {
+    "typescript": ">=4.7.2"
   }
 }
```

Comment:
**suggestion:** This would warn the user, if he installs the package with a Node.js version lower than 14.

See https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines

```suggestion
  },
  "engines": {
    "node": ">=14"
  }
}
```